.github/workflows/update.yml

```
@@ -0,0 +1,89 @@
+name: Update vuln-list
+on: [push, pull_request]
+
+jobs:
+  update:
+    name: Update repo vuln-list
+    runs-on: ubuntu-latest
+    env:
+      VULNLIST_REPOSITORY_OWNER: $(echo "$GITHUB_REPOSITORY" | awk -F / '{print $1}')
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+    steps:
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.14
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Setup github user email and name
+      run: |
+        git config --global user.email "action@github.com"
+        git config --global user.name "GitHub Action"
+
+    - name: NVD
+      run: go run main.go -target nvd
```

These steps are stateful, so we might be able to compile the binary and use it in later steps.

```suggestion
    - name: Compible vuln-list-update
      run: go install

   - name: NVD
      run: vuln-list-update -target nvd
```

or

```suggestion
    - name: Compible vuln-list-update
      run: go build -o vuln-list-update

   - name: NVD
      run: ./vuln-list-update -target nvd
```